src/main/java/ploopy/PloopyException.java

```diff
@@ -0,0 +1,14 @@
+package ploopy;
+
+/**
+ * An exception class that encapsulates the error message during
+ * the execution of the program.
+ */
+public class PloopyException extends Exception {
+
+    PloopyException(String message) {
```

Comment:
Could consider making child-classes of PloopyException to further specify the type of error.